isis/src/base/objs/OverlapNormalization/OverlapNormalization.cpp

```diff
@@ -264,7 +279,7 @@ namespace Isis {
         input.resize(m_statsList.size());
         for (int i = 0; i < (int)input.size(); i++) input[i] = 0.0;
         input[hold] = 1.0;
-        m_gainLsq->AddKnown(input, 0.0, 1e30);
+        m_gainLsq->AddKnown(input, 0.0, 1e10);
```

Comment:
These two changes are required to make the new SuperLU happy. The design matrix for these overlaps are all 0, 1, -1. Using something like 1E30 makes the reciprocal of the condition number too small and SuperLU complains about it being nearly singular. Changing to 1E10 will still keep these images held, but makes sure that the matrix isn't too close to singular.